e2e/e2eutil/deployments.go

```diff
@@ -36,3 +36,20 @@ func WaitForLastDeploymentStatus(jobID, status string, wc *WaitConfig) error {
 	})
 	return err
 }
+
+func LastDeploymentID(jobID string) (string, error) {
+	out, err := Command("nomad", "deployment", "list")
+	if err != nil {
+		return "", fmt.Errorf("could not get deployment list: %v\n%v", err, out)
+	}
+	rows, err := ParseColumns(out)
```

Comment:
:TIL: about ParseColumns - wish it's this simple in bash...

Reply:
We only landed it a couple weeks ago! 😀